weldx/asdf/tags/weldx/time/time.py

```diff
@@ -0,0 +1,27 @@
+from weldx.asdf.types import WeldxType
+from weldx.time import Time
+
+__all__ = ["TimeType"]
+
+
+class TimeType(WeldxType):
+    """A simple implementation of serializing a Time instance."""
+
+    name = "time/time"
+    version = "1.0.0"
+    types = [Time]
+
+    @classmethod
+    def to_tree(cls, node: Time, ctx):
+        """Serialize timedelta to tree."""
+        tree = dict()
+        tree["values"] = node.as_pandas()
+        tree["reference_time"] = node._time_ref
```

Comment:
```suggestion
        tree["values"] = node.as_timedelta()
        tree["reference_time"] = node.reference_time
```
I think it is cleaner this way. We used this structure before with our xarrays and there is no need to store absolute arrays if we have a reference time.

Reply:
true, however storing the datetime-values would enable direct comparison of the min/max values from the schema (for validation or others) which I kind of like even though we don't have these kind of validators implemented yet

One more problem I see with the above implementation (or the `as_timedelta` function in general) is that it would not reconstruct a `DatetimeIndex` into the same time object because of setting an explicit reference time.

```python
t1 = Time(["2020", "2021"])
t2 = Time(t1.as_timedelta(), t1.reference_time)

assert t1._time_ref is None
assert t2._time_ref is not None

t1.equals(t2) # False
```